assignment4/devisionUSingPromise.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Title</title>
+        <script>
+            function getPromise() {
+                let number1 = parseInt(document.getElementById('number1').value);
```

Comment:
This function is tightly coupled with inputs "number1" and "number2". please remove tightly coupling.